src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -35,6 +35,10 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
       BronzeTargets.jobRunsSnapshotTarget
     )
   }
+  def targetToSnap: Array[PipelineTable] = {
```

Comment:
Since we're moving all the logic from Bronze to a new class Snapshot we should drop all the logic from Bronze so we can stop maintaining it. For backwards compatibility simply reference the appropriate snapshot apply method to create a full (batch) snapshot so that it continues to work and add the @deprecated decorator.